src/main/java/org/apache/pulsar/io/jcloud/format/ParquetFormat.java

```diff
@@ -76,7 +76,12 @@ public void configure(BlobStoreAbstractConfig configuration) {
         this.useMetadata = configuration.isWithMetadata();
         this.useHumanReadableMessageId = configuration.isUseHumanReadableMessageId();
         this.useHumanReadableSchemaVersion = configuration.isUseHumanReadableSchemaVersion();
-        this.compressionCodecName = CompressionCodecName.fromConf(configuration.getParquetCodec());
+        if (configuration.getParquetCodec() == null || configuration.getParquetCodec().isEmpty()
+                || configuration.getParquetCodec().equals("none")) {
```

Comment:
It's better to put this logic to `BlobStoreAbstractConfig`. So that we can maintain the default value in the Config class.